packages/utils/src/lib/string.ts

```diff
@@ -0,0 +1,58 @@
+import type { CamelCaseToKebabCase } from './types.js';
+
+/**
+ * Converts a kebab-case string to camelCase.
+ * @param string - The kebab-case string to convert.
+ * @returns The camelCase string.
+ */
+export function kebabCaseToCamelCase(string: string) {
```

Comment:
Since the inverse function uses a `CamelCaseToKebabCase<T>`, why not create a `KebabCaseToCamelCase<T>` type also? It's actually a lot easier to do the conversion in this direction:

```ts
export type KebabCaseToCamelCase<T extends string> =
  T extends `${infer First}-${infer Rest}`
    ? `${First}${Capitalize<KebabCaseToCamelCase<Rest>>}`
    : T;
```

Reply:
Thanks! added and used